unit-tests/src/test/scala/java/util/PropertiesTest.scala

```diff
@@ -358,32 +349,51 @@ class PropertiesTest {
                  prop.store(null: OutputStream, ""))
   }
 
+  def store(props: Properties, header: String = ""): ByteArrayOutputStream = {
+    val out = new ByteArrayOutputStream()
+    props.store(out, header)
+    out.close()
+    out
+  }
+
+  def storeWriter(props: Properties,
+                  header: String = ""): ByteArrayOutputStream = {
+    val out = new ByteArrayOutputStream()
+    props.store(new OutputStreamWriter(out), header)
+    out.close()
+    out
+  }
+
+  def load(out: ByteArrayOutputStream): Properties =
+    load(new ByteArrayInputStream(out.toByteArray))
+
+  def load(in: ByteArrayInputStream): Properties = {
+    val props = new Properties()
+    props.load(in)
+    in.close()
+    props
+  }
+
+  def assertAll(props1: Properties, props2: Properties): Unit = {
+    val e = props1.propertyNames()
+    while (e.hasMoreElements) {
+      val nextKey = e.nextElement().asInstanceOf[String]
+      assertEquals(props2.getProperty(nextKey), props1.getProperty(nextKey))
+    }
+  }
+
   // used for next two tests, \b prints as \u0008
   val prop1 = new Properties()
```

Comment:
This now only used in `store_OutputStream_comments_load_InputStream_roundtrip()`. Consider moving it inside that method, along with `out1`. And then redeclare another `out1` based on some trivial `Properties` in the test `check_comment_formatted_correctly()` (or merge that test with the previous one to avoid even that).

Reply:
I fixed most everything I could see up. I also made the test names consistent with toggle case.